collectors/python.d.plugin/sensors/metadata.yaml

```
@@ -1,54 +1,127 @@
-    link: ''
+    link: https://hwmon.wiki.kernel.org/lm_sensors
     categories:
-    - data-collection.hardware-devices-and-sensors
-    icon_filename: 'microchip.svg'
+      - data-collection.hardware-devices-and-sensors
+    icon_filename: "microchip.svg"
   related_resources:
     integrations:
       list: []
   info_provided_to_referring_integrations:
-    description: ''
-  keywords: []
+    description: ""
+  keywords:
+    - sensors
+    - temperature
+    - voltage
+    - current
+    - power
+    - fan
+    - energy
+    - humidity
   most_popular: false
 overview:
   data_collection:
-    metrics_description: 'Examine Linux Sensors metrics with Netdata for insights into hardware health and performance. Enhance your system''s reliability with real-time hardware health insights.'
-    method_description: ''
+    metrics_description: >
+      Examine Linux Sensors metrics with Netdata for insights into hardware health and performance.
+
+      Enhance your system's reliability with real-time hardware health insights.
+    method_description: >
+      Reads system sensors information (temperature, voltage, electric current, power, etc.) via [lm-sensors](https://hwmon.wiki.kernel.org/lm_sensors).
   supported_platforms:
     include: []
     exclude: []
   multi_instance: true
   additional_permissions:
-    description: ''
+    description: ""
   default_behavior:
     auto_detection:
-      description: ''
+      description: >
+        The following type of sensors are auto-detected:
+
+        - temperature
+        - fan
+        - voltage
+        - current
+        - power
+        - energy
+        - humidity
     limits:
-      description: ''
+      description: ""
     performance_impact:
-      description: ''
+      description: ""
 setup:
   prerequisites:
     list: []
   configuration:
     file:
-      name: ''
-      description: ''
+      name: python.d/sensors.conf
+      description: ""
     options:
-      description: ''
+      description: |
+        There are 2 sections:
+
+        * Global variables
+        * One or more JOBS that can define multiple different instances to monitor.
+
+        The following options can be defined globally: priority, penalty, autodetection_retry, update_every, but can also be defined per JOB to override the global values.
+
+        Additionally, the following collapsed table contains all the options that can be configured inside a JOB definition.
+
+        Every configuration JOB starts with a `job_name` value which will appear in the dashboard, unless a `name` parameter is specified.
       folding:
-        title: ''
+        title: Config options
         enabled: true
-      list: []
+      list:
+        - name: types
+          description: The types of sensors to collect.
+          default_value: >
+            - temperature
+            - fan
+            - voltage
+            - current
+            - power
+            - energy
+            - humidity
```

actually here you need a way to render this in a cell `>` would remove newlines, resulting in `- temperature - fan` etc
```suggestion
          default_value: "temperature, fan, voltage, current, power, energy, humidity"
```
this would make most sense I think